syntax = "proto3";
import "Rectangle.proto";
package streetgangpb;


message GetSceneRequest
{
	string track_id = 1;
	uint64 world_id = 2;
	Rectangle rect = 3;
}
